src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -485,6 +570,90 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+              CASE('ZONEHVAC:TERMINALUNIT:VARIABLEREFRIGERANTFLOW')
+                isVariableVolume = .FALSE.
+                CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                nodiff = .false.
+                OutArgs(1:13) = InArgs(1:13)
+                IF (SameString(InArgs(14), 'FAN:VARIABLEVOLUME')) THEN
+                  isVariableVolume = .TRUE.
+                  OutArgs(14) = 'Fan:SystemModel'
+                  OutArgs(15) = TRIM(InArgs(15))
+                  sysFanName = TRIM(InArgs(15))
+                ELSE
+                  OutArgs(14:15) = InArgs(14:15)
+                ENDIF
+                OutArgs(16:CurArgs) = InArgs(16:CurArgs)
+                CALL WriteOutIDFLines(DifLfn, 'ZoneHVAC:TerminalUnit:VariableRefrigerantFlow', CurArgs, OutArgs, NwFldNames, NwFldUnits)
+
+                IF (isVariableVolume) THEN
+                  ! create fan system model object
+                  ObjectName = 'Fan:SystemModel'
+                  DO Num3 = 1, NumFanVariableVolume
+                    IF (SameString(OldFanVO(Num3)%oldFanName, sysFanName)) THEN
+                      CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                      OutArgs(1) = TRIM(sysFanName)
+                      OutArgs(2) = OldFanVO(Num3)%availSchedule
+                      OutArgs(3) = OldFanVO(Num3)%inletAirNodeName
+                      OutArgs(4) = OldFanVO(Num3)%outletAirNodeName
+                      OutArgs(5) = OldFanVO(Num3)%maxAirFlow_str
+                      OutArgs(6) = 'Continuous'                             !- Speed Control Method
+                      IF (SameString(OldFanVO(Num3)%minFlowInputMethod, "FixedFlowRate")) THEN
+                        IF (.NOT. SameString(OldFanVO(Num3)%maxAirFlow_str, "AUTOSIZE")) THEN
+                          READ(OldFanVO(Num3)%fanPowerMinAirFlow_str, '(F15.5)') fanPowerMinAirFlow
+                          READ(OldFanVO(Num3)%maxAirFlow_str, '(F15.5)') maxAirFlow
+                          WRITE(OutArgs(7), '(F15.5)') (fanPowerMinAirFlow / maxAirFlow)
+                      ELSE ! maxAirFlow_stris autosize
+                          READ(OldFanVO(Num3)%fanPowerMinAirFlow_str, '(F15.5)') fanPowerMinAirFlow
+                          IF (.NOT. fanPowerMinAirFlow == 0) THEN ! don't know how to do division with autosize
+                            CALL writePreprocessorObject(DifLfn, PrognameConversion, 'Warning', &
+                                    'Cannot calculate Electric Power Minimum Flow Rate Fraction for' // sysFanName // &
+                                            'when Maximum Flow Rate is autosize and Fan Power Minimum Air Flow Rate is non-zero')
```

Comment:
This should also call `ShowWarningError` so that the transition audit will show there was an issue.
Writing the preprocessor object also is good so that the warning will stay with the idf file, but I would add more context, something like:
Cannot calculate Electric Power Minimum Flow Rate Fraction for Fan:SystemModel=' // sysFanName // &
                                            'when old Fan:VariableVolume Maximum Flow Rate is autosize and Fan Power Minimum Air Flow Rate is non-zero'
Electric Power Minimum Flow Rate Fraction is set to zero.

Reply:
Good point. I added the ShowWarningError and changed the error message content to provide more info.